pkgs/applications/misc/gcalcli/default.nix

```diff
@@ -1,5 +1,10 @@
-{ stdenv, lib, fetchFromGitHub, python3
-, libnotify ? null }:
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  python3,
+  libnotify ? null,
+}:
 
 with python3.pkgs;
```

Comment:
```suggestion
```

Reply:
Thanks for all the review! I noticed other packages using `python3Packages` instead; is that preferred?

Reply:
It's the only thing that supports cross compilation.